stdlib/2and3/colorsys.pyi

```diff
@@ -0,0 +1,10 @@
+# Stubs for colorsys
+
+from typing import Tuple
+
```

Comment:
These are undocumented, seems internal (not in `__all__`, nor having description); it was added in fc55ed0 if really wanted.